### PR TITLE
Fix: Changed the visible_tooltip_attribute to a list format

### PR DESCRIPTION
**Pull request by Geeker1**
## Description
Changed the visible_tooltip _attribute JSONField to use a list not dict

## Related Issue
#248 
## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Review by milafrerichs — CHANGES_REQUESTED** [review]
can you add the migration that is needed alongside and also create a data migration?

**Comment by Geeker1** [comment]
> can you add the migration that is needed alongside and also create a data migration?

Ok

**Comment by Geeker1** [comment]
> can you add the migration that is needed alongside and also create a data migration?

Migration script has been created, local migration to DB worked.

**Comment by milafrerichs** [comment]
great, can you also create a data migration? to update all existing instances? thanks.

**Comment by Geeker1** [comment]
> great, can you also create a data migration? to update all existing instances? thanks.

Done, although I included only the values from the dict, not sure if the key is needed too.

File: wazimap_ng/points/migrate_tooltip_data.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+
+from wazimap_ng.points.models import ProfileCategory
+
+def migrate_attributes():
```
Comment (milafrerichs) [inline wazimap_ng/points/migrate_tooltip_data.py:5]: I meant a data **migration**

Create a new migration file and update the field there. 

it should look like this:
https://github.com/OpenUpSA/wazimap-ng/compare/chore/data-migration#diff-b5add147a76fbb22069454c268cf5e84368d0f14b12e921b3891192440ef50c6

**Review by milafrerichs — CHANGES_REQUESTED** [review]

**Review by milafrerichs — APPROVED** [review]

**Pull request merged**